src/util/navigation.js

```diff
@@ -180,6 +184,15 @@ export function newMessageUrl () {
   return `${messagesUrl()}/new`
 }
 
+export function messageThreadUrl (person) {
+  // TODO: messageThreadId doesn't seem to be currently ever coming-in from the backend
```

Comment:
what does this do on the front-end? file a ticket?

Reply:
https://github.com/Hylozoic/hylo-evo/issues/655